### PR TITLE
Live model card service

### PR DESCRIPTION
**Pull request by habdelra**
This PR introduces a `CardService.loadLiveModel()` (which is a sibling of `CardService.loadModel()` which has been renamed to `loadStaticModel()`). Also code-mode has been updated to use this new service. In a subsequent PR i'll incorporate this service in interact mode as well.

### CODE REVIEW

File: packages/host/tests/acceptance/operator-mode-test.ts [hunk 1]
```diff
@@ -1132,7 +1143,15 @@ module('Acceptance | operator mode tests', function (hooks) {
...
     });
 
-    setMonacoContent(JSON.stringify(expected));
+    await this.expectEvents(
+      assert,
+      realm,
+      adapter,
+      expectedEvents,
+      async () => {
+        setMonacoContent(JSON.stringify(expected));
```
Comment (backspace) [inline packages/host/tests/acceptance/operator-mode-test.ts:1152]: This diff reads a bit weirdly to me, there‘s an expectation but the callback doesn’t seem to relate to it? Could the callback be optional?

**Review by backspace — APPROVED** [review]
The code looks good to me, I wasn’t able to exercise it in the preview environment nor locally but I think for the latter it’s probably something weird on my end, not a problem with the PR. I’ll try again tomorrow.

File: packages/host/tests/acceptance/operator-mode-test.ts (hunk 1 above)
Comment (habdelra) [reply packages/host/tests/acceptance/operator-mode-test.ts:1152]: The diff is that I added the `expectedEvents()` which internally has a test assertion (which is why it was increased). The callback is absolutely necessary—it’s what triggers the auto save

Comment (backspace) [reply packages/host/tests/acceptance/operator-mode-test.ts:1152]: 👍🏻 understood, thanks

**Comment by backspace** [comment]
I’m still unable to save a card locally but again unsure it’s actually a problem with this PR. The errors:

Boxel 2023-10-12 16-00-32

**Pull request merged**